### ISSUE TITLE
Create OnBoarding screen for iOS

### ISSUE THREAD

**Issue by Odrakir** [toggl/mobileapp#6999]
Do it according to design and bind it with the new view model.

Be aware of errors and show them when appropriate.

#6998 should be done first.

Designs and full specs: https://www.notion.so/toggl/Designs-wip-d767ebc157c542c790e1bc93eeaf006f#835299ea40344044b2572f356b4f47c1

**NOTE:** This issue is part of the `Onboarding refactor` project and it should be merged into the project branch and not into develop.
